addons/web/static/src/js/views/kanban/kanban_renderer.js

```diff
@@ -276,14 +277,24 @@ var KanbanRenderer = BasicRenderer.extend({
             previousCard.focus();
         }
     },
+    /**
```

Comment:
Can you change to:

```
/**
 * Returns the default columns for the kanban view example background.
 *
 * @private
 */
```

And also put the method *after* the "_renderExampleBackground" method?

Reply:
You can also add in a new line:

"You can override this method to easily customize the column names."

Reply:
@jip-odoo Actually I wanted the two sentences so like this:

```
/**
 * Returns the default columns for the kanban view example background.
 * You can override this method to easily customize the column names.
 *
 * @private
 */
```

Sorry for the misunderstanding.

Reply:
Changes done.